Blackjack.py

```diff
@@ -0,0 +1,282 @@
+#This code is based on the Udemy course milestone project
+#This code will take the Class and Card classes to make a 52 card deck
+#Will ask the player to make a bet agains the dealer and then play blackjack
+#Player will be asek to hit or take another card
+#This is a bit unique as I forgot how to play Blackjack and so the program keeps playing
+#until all 52 cards are used up
+
+import random
+
+#These are my constants for the future I will UPPER_SNAKE_CASE
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
+ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')
+values = {'Two':2, 'Three':3, 'Four':4, 'Five':5, 'Six':6, 'Seven':7, 'Eight':8, 'Nine':9, 'Ten':10, 'Jack':10,
+         'Queen':10, 'King':10, 'Ace':11}
+
+if __name__=="__main__":
+
+    class Card():
+
+        def __init__(self, suit, rank):
+            self.suit = suit
+            self.rank = rank
+            self.value = values[rank]
+
+        def __str__(self):
+            return f"{self.rank} of {self.suit}"
+
+
+    class Deck():
+
+        def __init__(self):
+            self.all_cards = []
+            for suits_card in suits:
+                for rank_card in ranks:
+                    self.all_cards.append(Card(suits_card, rank_card))

```

Comment:
For code re-usability, it might be useful to move this part to a function like `reset` or `build`.  Then your `Deck` object can be used again later. (Assuming the program is designed to clear out any old cards that could be floating around, but that's a different feature/topic). Not really needed for this program/script, just something to point out.

Reply:
I will reference this later. Interesting how Pycharm suggested the pep8 fix.